src/backend_manager.cc

```diff
@@ -192,16 +192,16 @@ TritonBackend::LoadBackendLibrary(
     std::unique_ptr<SharedLibrary> slib;
     RETURN_IF_ERROR(SharedLibrary::Acquire(&slib));
 
-    std::wstring original_path;
+    void* addDepDirCookie;
```

Comment:
looks like other local variables use snake_case instead of camelCase. 

please fix up.